docs/source/integrations/databricks_development_workflow.md

```diff
@@ -0,0 +1,324 @@
+
+```python
+git remote add origin <your-git-repository-url>
+git add .
+git commit -m "Initial commit"
+git push -u origin main
+```
+
+Replace `<your-git-repository-url>` with the URL of the Git repository you created earlier.
+
+### Set up Databricks Repos and clone your project
+
+The next step is to sync the remote repository stored on your Git provider to Databricks Repos.
+
+To integrate Databricks Repos with the private Git repository stored by your Git provider, you first need to obtain an access token from your Git provider. Access tokens authenticate an application that wishes to access your private Git repository.
+
+The process for obtaining a token varies depending on your Git provider. Follow the instructions for generating an access token and adding it to Databricks specific to your Git provider in the [Databricks documentation](https://docs.databricks.com/repos/get-access-tokens-from-git-provider.html).
+
+### Sync your Git repository with Databricks
+
+After obtaining your Git access token, sync your remote Git repository to Databricks:
+
+1. Navigate to the Repos tab in your Databricks workspace.
+2. Click on the 'Add Repo' button.
+3. In the 'Add Repo' dialogue, select your Git provider from the list.
+4. Enter the URL of your remote Git repository containing the Kedro project.
+5. Click the 'Add Repo' button to complete the setup.
+
+Find the connected Git repository and click on it in the Repos tab. Databricks will automatically clone the repository and display its contents. Your Kedro project is now available in Databricks Repos, and you can start working with it in the Databricks Workspace.
+
+```{note}
+Databricks Repos supports some Git operations that you can use to manage your repository, see the [Databricks documentation](https://docs.databricks.com/repos/git-operations-with-repos.html) for reference.
+```
+
+### Run your project using Databricks notebooks
+
+Now that your project is synced with Databricks Repos, the following sections will guide you through running it on a cluster using notebooks.
+
+#### Create a new Databricks notebook
+
+To run the Python code from your Databricks repo, [create a new Databricks Notebook](https://docs.databricks.com/notebooks/notebooks-manage.html#create-a-notebook) and attach it to your cluster.
+
+#### Install project dependencies from the requirements.txt file
+
+Before you import and run your Python code, you'll need to install your project's dependencies. Your project has a `requirements.txt` file for this purpose. Install the dependencies using the %pip magic command in a new cell in your notebook:
+
+```bash
+%pip install -r "/Workspace/Repos/<your_username>/iris-databricks/src/requirements.txt"
+```
+
+This command will install the dependencies listed in your requirements.txt file to the Databricks cluster attached to your notebook.
+
+#### Upload project data to DBFS
```

Comment:
Why is this needed? I got an error at this stage and just skipped and can still run my pipeline fine.

Reply:
You were likely using the same cluster as me, where I have already uploaded this data to DBFS as part of my own testing.

This would be easier to explain offline.